frontend/test/card_info_util_test.dart

```diff
@@ -41,5 +41,18 @@ void main() {
       final pepper = const Base64Decoder().convert("MvMjEqa0ulFDAgACElMjWA==");
       expect(cardInfo.hash(pepper), 'zogEJOhnSSp//8qhym/DdorQYgL/763Kfq4slWduxMg=');
     });
+
+    test("should be stable for a Nuernberg Pass with startDay", () {
+      final cardInfo = CardInfo()
+        ..fullName = 'Max Mustermann'
+        ..expirationDay = 365 * 40 // Equals 14.600
+        ..extensions = (CardExtensions()
+          ..extensionRegion = (RegionExtension()..regionId = 93)
+          ..extensionBirthday = (BirthdayExtension()..birthday = -365 * 10)
+          ..extensionNuernbergPassNumber = (NuernbergPassNumberExtension()..passNumber = 99999999)
+          ..extensionStartDay = (StartDayExtension()..startDay = 365 * 2));
+      final pepper = const Base64Decoder().convert("MvMjEqa0ulFDAgACElMjWA==");
+      expect(cardInfo.hash(pepper), '1ChHiAvWygwu+bH2yOZOk1zdmwTDZ4mkvu079cyuLjE=');
+    });
```

Comment:
i did not rerequest your review because i wasn't finished :D 
The card hash did not update in `administration` after adding extension so i removed the test temp.
I have problems running karma tests. So i decided to wait for sarahs prs.

Reply:
ok now everything works, gonna update soon :)

Reply:
added a test for administration

Reply:
> i did not rerequest your review because i wasn't finished :D

Ah sorry :D Was too curious :) Will wait next time